### PR TITLE
SB-16 Simulation integration and import overhaul

### PR DESCRIPTION
**Pull request by Ichtiostega**
Integrated simulation into environment, moved simulation directory into environment directory, fixed the imports in simulation and utils.

### CODE REVIEW

File: environment/simulation/simulation.py [hunk 1]
```diff
@@ -6,11 +6,16 @@
...
-from utils.generate_map import Difficulty
+try:
+    from .utils import simulation_utils as utils
+    from .utils import simulation_pymunk_utils as pymunk_utils
+    from .utils import simulation_pygame_utils as pygame_utils
+    from .utils.generate_map import Difficulty
+except ImportError:
+    import utils.simulation_utils as utils
+    import utils.simulation_pymunk_utils as pymunk_utils
+    import utils.simulation_pygame_utils as pygame_utils
```
Comment (KurekWioletta) [inline environment/simulation/simulation.py:17]: from utils import simulation_utils as utils? :)

File: environment/swarmball_env.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import gym
 from gym import spaces, logger
 from gym.utils import seeding
-from simulation.simulation import SwarmBallSimulation
+from .simulation.simulation import SwarmBallSimulation
```
Comment (KurekWioletta) [inline environment/swarmball_env.py:4]: try catch {}

**Review by Grzybu1 — APPROVED** [review]
Nieźle byq

File: environment/simulation/simulation.py (hunk 1 above)
Comment (Ichtiostega) [reply environment/simulation/simulation.py:17]: Jaka jest różnica?

File: environment/swarmball_env.py (hunk 2 above)
Comment (Ichtiostega) [reply environment/swarmball_env.py:4]: Nie planowałem tego używać jako standalone dlatego nie ma try except.

**Review by TheMoira — APPROVED** [review]

File: environment/simulation/simulation.py (hunk 1 above)
Comment (Heecatee) [reply environment/simulation/simulation.py:17]: Consistency :D

Comment (Heecatee) [reply environment/simulation/simulation.py:17]: Consistency :D

Comment (Heecatee) [reply environment/simulation/simulation.py:17]: Consistency :D

Comment (Heecatee) [reply environment/simulation/simulation.py:17]: Consistency :D

**Review by Heecatee — APPROVED** [review]

**Pull request merged**